components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit-finish-local-ajaxprocessor.jsp

```diff
@@ -111,6 +111,27 @@
         }
         property.setValue(samlMetadataSigningEnabled);
         propertyList.add(property);
+
+        property = new Property();
+        property.setName(IdentityApplicationConstants.Authenticator.SAML2SSO.SAML_METADATA_WANT_AUTHN_REQUESTS_SIGNED_ENABLED);
+        String WantAuthnRequestsSignedEnabled = "false";
+        if (StringUtils.containsIgnoreCase(request.getParameter("WantAuthnRequestsSigned"), "on")) {
+                    WantAuthnRequestsSignedEnabled = "true";
+                }
+        property.setValue(WantAuthnRequestsSignedEnabled);
+        propertyList.add(property);
+
+
+        property = new Property();
+        property.setName(IdentityApplicationConstants.Authenticator.SAML2SSO.SAML_METADATA_NAME_ID_FORMAT);
+        property.setValue(request.getParameter("NameIDFormat"));
+        propertyList.add(property);
+
+        property = new Property();
+        property.setName(IdentityApplicationConstants.Authenticator.SAML2SSO.SAML_METADATA_ATTRIBUTE_PROFILE);
+        property.setValue(request.getParameter("AttributeProfile"));
```

Comment:
Let's use the previously defined constants for "NameIDFormat" and "AttributeProfile"